web/cobrands/sass/_base.scss

```diff
@@ -182,6 +182,11 @@ a {
   }
 }
 
+.meta-2 a[name] {
```

Comment:
You moved the `<a>` outside of `<p class="meta-2">` in [update.html](https://github.com/mysociety/fixmystreet/blob/ca10ef870ca9525d3223fe0cfe9c1c78228cd3d6/templates/web/base/report/update.html) but it looks like you forgot to update this selector to match the new position.

Rather than concocting some selector like `item-list__update-wrap > a[name]` I would probably just add a class to the `<a>` and use that.